README.md

```diff
@@ -71,42 +79,82 @@ CREATE EXTENSION vectors;
 We support three operators to calculate the distance between two vectors:
 
 - `<->`: square Euclidean distance
-- `<#>`: dot product distance
-- `<=>`: cosine distance
+- `<#>`: negative dot product distance
+- `<=>`: negative square cosine distance
 
 ```sql
 -- call the distance function through operators
 
 -- square Euclidean distance
-SELECT array[1, 2, 3] <-> array[3, 2, 1];
+SELECT '[1, 2, 3]' <-> '[3, 2, 1]';
```

Comment:
Does '[1,2,3]' mean a string here? Will there be extra parsing cost?

Reply:
`'[1, 2, 3]'` will be casted to `vector`, since operands of `<->` are `vector`.

Yes, it will. A cast function from an array to `vector` might be helpful.